src/handlers/motions/helpers.ts

```diff
@@ -1,6 +1,7 @@
 import { BigNumber } from 'ethers';
```

Comment:
Has this file got a different purpose than `utils/motions`, ie. is used by different parts of the ingestor? Otherwise I'd consider merging them or keeping them next to each other, in whatever way you think makes sense